crates/bitwarden-json/src/command.rs

```diff
@@ -1,21 +1,29 @@
-use bitwarden::{
-    auth::request::AccessTokenLoginRequest,
-    secrets_manager::{
-        projects::{
-            ProjectCreateRequest, ProjectGetRequest, ProjectPutRequest, ProjectsDeleteRequest,
-            ProjectsListRequest,
-        },
-        secrets::{
-            SecretCreateRequest, SecretGetRequest, SecretIdentifiersRequest, SecretPutRequest,
-            SecretsDeleteRequest,
-        },
+use bitwarden::auth::request::AccessTokenLoginRequest;
```

Comment:
Access token is a secrets manager concept.

Reply:
It's hidden under the feature flag now.